### ISSUE TITLE
Cant test recognition image

### ISSUE THREAD

**Issue by blennerSilva** [Qualeams/Android-Face-Recognition-with-Deep-Learning-Test-Framework#30]
Hello i just recentrly dowloaded your demo to test but after i add a new person i cant do anything with it because de buttons are not enabled ...how do i get to enable the whole app?

**Comment by sladomic** [comment]
Hello

- Try to close the app completely and restart it
- Check that the person has been added in /sdcard/Pictures/facerecognition/training as a new folder

The Training button gets enabled if you have added at least one person to the training set

The same for the Test button and the test set

Regards,
Michael

**Comment by blennerSilva** [comment]
does it only work if i have an sdcard ?

**Comment by sladomic** [comment]
No, /sdcard is the built in sdcard
